readme.md

```diff
@@ -23,43 +27,50 @@ PHARE is an open-source projet licenced under the GPLv3. Please refer to [LICENC
 
 ## Install
 
-For system library requirements see the file:
-  https://github.com/PHARCHIVE/phare-teamcity-fc31/blob/master/Dockerfile
+For system library requirements see the following [Docker File](https://github.com/PHARCHIVE/phare-teamcity-fc31/blob/master/Dockerfile)
```

Comment:
I renamed this repo today but https://github.com/PHARCHIVE/phare-teamcity-agent/blob/master/Dockerfile

but it should forward if it's not updated